CHANGELOG.md

```diff
@@ -13,6 +13,9 @@
 * [CHANGE] Deprecated `-distributor.limiter-reload-period` flag. #1766
 * [CHANGE] Ingesters now write only normalised tokens to the ring, although they can still read denormalised tokens used by other ingesters. `-ingester.normalise-tokens` is now deprecated, and ignored. If you want to switch back to using denormalised tokens, you need to downgrade to Cortex 0.4.0. Previous versions don't handle claiming tokens from normalised ingesters correctly. #1809
 * [CHANGE] Overrides mechanism has been renamed to "runtime config", and is now separate from limits. Runtime config is simply a file that is reloaded by Cortex every couple of seconds. Limits and now also multi KV use this mechanism.<br />New arguments were introduced: `-runtime-config.file` (defaults to empty) and `-runtime-config.reload-period` (defaults to 10 seconds), which replace previously used `-limits.per-user-override-config` and `-limits.per-user-override-period` options. Old options are still used if `-runtime-config.file` is not specified. This change is also reflected in YAML configuration, where old `limits.per_tenant_override_config` and `limits.per_tenant_override_period` fields are replaced with `runtime_config.file` and `runtime_config.period` respectively. #1749
+* [CHANGE] Moved MinChunkAge to the Querier config as QueryStoreAfter, allowing the store to be skipped during query time if the metrics wouldn't be found.#1893 
```

Comment:
`MinChunkAge` and `QueryStoreAfter` are internal names and not the names of the config options. I would replace them with the CLI flags, to make them easier to understand for the final user.

Please also mention that the `ingestermaxquerylookback` YAML option has been renamed to `query_ingesters_within` (in case anyone is configuring it through YAML).

Reply:
done.